topics/assembly/metadata.yaml

```diff
@@ -63,11 +57,39 @@ material:
       - "We learned about the strategies that assemblers use to make reference genomes"
       - "We performed a number of assemblies with Velvet and SPAdes."
       - "You should use SPAdes or another more modern assembler than Velvet for actual assemblies now."
-    contributors:
-      - slugger70
+  -
+    title: "Unicycler Assembly"
+    type: "tutorial"
+    name: "unicycler-assembly"
+    zenodo_link: "https://doi.org/10.5281/zenodo.842795"
+    galaxy_tour: "yes"
+    hands_on: "yes"
+    slides: "yes"
+    questions:
+      - "How to perform an hybrid bacterial assembly with Unicycler"
+    objectives:
+      - "Evaluate Illumina read quality"
+      - "Perform a Small genome Assembly with Unicycler"
+      - "Evaluate the Quality of the Assembly with Quast"
+      - "Annotate the assembly with Prokka"
+    time_estimation: "4h"
+    key_points:
 
 maintainers:
-  - slugger70
```

Comment:
We put usernames only here now and there is a CONTRIBUTORS file with all the information.

Reply:
so, `contributors` goes away from this file?

Reply:
it stays in, but only the username of the contributor/maintainer should be given (like this removed line), not the full name/email etc like it used to be. This name is then matched to the information in the `CONTRIBUTORS.yaml` file in the root of the repo, where information such as full name and email address can be associated with the user